phivk/gradu.al
# Feature/calendar links and events

This PR adds two features:
- A Google calendar link
- A dynamic ICS file generation and download

To test:
- Navigate to an event
- Click on the Google Calendar event
- Confirm that the details match
- Click on the download ICS button
- Use a tool such as this one https://icalendar.org/validator.html#results to check that it is valid
